lib/BenchShow/Common.hs

```diff
@@ -81,6 +87,28 @@ filterSanity label old new = do
         ++ "\nOriginal groups: " ++ show old
         ++ "\nNew groups: " ++ show new
 
+type CSV = [[String]]
+
+-- XXX This is ugly in performance but works for the time being.
+-- XXX This lib should not depend on internal modules of streamly.
+parseCSVFromFile :: FilePath -> IO CSV
+parseCSVFromFile inFile = do
+    src <- IO.openFile inFile ReadMode
+
+    Stream.unfold Handle.read src -- SerialT IO Word8
+        & Unicode.decodeUtf8      -- SerialT IO Char
+        & parseLines              -- IO CSV
+
+    where
+
+    parseLine ls =
+        Stream.toList $ Stream.splitOn (== ',') Fold.toList $ Stream.fromList ls
```

Comment:
This won't work with quoted fields that include escaped commas. We had frame parsing in streamly for that use case, but not sure if we have implemented it yet.